alcs-api/src/application-reconsideration/application-reconsideration.module.ts

```diff
@@ -0,0 +1,24 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ApplicationModule } from '../application/application.module';
+import { BoardModule } from '../board/board.module';
+import { CardModule } from '../card/card.module';
+import { CodeModule } from '../code/code.module';
+import { ReconsiderationProfile } from '../common/automapper/reconsideration.automapper.profile';
+import { ApplicationReconsiderationController } from './application-reconsideration.controller';
+import { ApplicationReconsideration } from './application-reconsideration.entity';
+import { ApplicationReconsiderationService } from './application-reconsideration.service';
+
+@Module({
+  imports: [
+    TypeOrmModule.forFeature([ApplicationReconsideration]),
```

Comment:
Shouldn't ApplicationReconsiderationType also be here?

Reply:
I'm not using it for anything at this point so it does not really matter. It is in CodeModule where the fetching happens